### ISSUE TITLE
Does Wrapper work with Managed Accounts?

### ISSUE THREAD

**Issue by franchiceddhu** [aunefyren/wrapperr#102]
As title says. I use several Managed Accounts at home, yet wrapper only fetches data from the main one (which is empty as I use it for admin purposes only).

Is this another limit of Managed Accounts? Shame on you, Plex!

**Comment by franchiceddhu** [comment]
Thanks again for your help! 

So, I've disabled Plex Auth. It looks like I can then log just by using the Managed Account username (on wrapper's users page, I can see every one of them with their respective IDs). It succesfully logs in whenever I input a correct, existing username. However, it will still show no shows watched for either one of my Managed users. 

The timeframe is set correctly.

**Comment by aunefyren** [comment]
Can you confirm that Tautulli says that the managed users have watched shows?

**Comment by franchiceddhu** [comment]
Hey, thanks again. Tautulli works flawlessly. Here's a screenshot for a managed user:

![Screenshot_20241229_172248~2](https://github.com/user-attachments/assets/99c70289-fb50-4bb0-8318-73a72d5c7cdf)

Here's a screenshot from the wrapper for that very same user (on the other fields, it reports 0 movies and 0 tv shows watched as well):

![Screenshot_20241229_172513_Chrome](https://github.com/user-attachments/assets/1955012b-e856-49b9-94dd-8130041d0eb8)

I don't know if it's useful, but the only non-managed user, aka the main one, is used only for administration purposes and therefore has no movies/shows played on it (I, the admin, have my own managed user that I use to watch any media).

**Comment by franchiceddhu** [comment]
Ok, strange as it sounds - as soon as I watched a TV episode on my main non-managed account, and after clearing cache and re-syncing users from Tautulli, looks like the problem solved by itself. I am able to sign in with every managed user's name, and get the correct stats both for every user and server-wide. 

I tried to reproduce the issue by removing the Tautulli history for the non-managed account, yet Wrapper still worked afterwards, so it's like the first time I watched a content from the non-managed account it "unlocked" everything.

Thanks for your support anyways, and your really great application. Have a nice 2025!
